prepare-mac-os.sh

```diff
@@ -22,7 +22,13 @@ elif [ -e "/usr/local/bin/brew" ]; then
 	sudo -u $CURRENT_USER brew install libelf
 	sudo -u $CURRENT_USER brew install libmpc
 	sudo -u $CURRENT_USER brew install libusb
-
+    sudo -U $CURRENT_USER brew install sdl
```

Comment:
Sdl should not be needed

Reply:
I haven't used this repo in a long time so I don't remember the reasoning for adding it. I'm guessing something mentioned an SDL requirement but I'll see if I can get the toolchain installed without it.